src/app/job/[id]/page.module.css

```
@@ -0,0 +1,72 @@
+.lists {
+  display: grid;
+  gap: 1rem;
+  padding: 1rem;
+  border: 1px solid transparent;
+  border-bottom-color: var(--neutral-shade-90);
+
+  &:hover {
+    border-color: var(--neutral-shade-60);
+    border-radius: var(--border-radius-square);
+    background-color: var(--primary-tint-02);
+  }
+
+  header {
```

پیشنهاد میکنم واسه اینجور تگ‌ها که معمولاً المان‌های زیادی داخلشون هست، از Child Combinator استفاده کنید
چون احتمال این هست که تو یکی از المان‌های داخلی باز دوباره از این استفاده کرده باشید و استایل‌هاتون با هم تداخل پیدا کنه

```suggestion
  > header {
```

---

اصلاح شد